roseau/load_flow/tests/test_electrical_network.py

```diff
@@ -31,147 +30,80 @@
-                    [20000.0, 2.891203383964549e-18],
-                    [-10000.000000000002, -17320.508076],
-                    [-10000.000000000002, 17320.508076],
-                    [-1.3476481215690672e-12, 2.891203383964549e-18],
-                ],
-            },
-            {
-                "id": "bus1",
-                "phases": "abcn",
-                "potentials": [
-                    [19999.949999875, 2.8911961559741588e-18],
-                    [-9999.974999937502, -17320.464774621556],
-                    [-9999.974999937502, 17320.464774621556],
-                    [0.0, 0.0],
-                ],
-            },
-        ],
-        "branches": [
-            {
-                "id": "line",
-                "phases1": "abcn",
-                "phases2": "abcn",
-                "currents1": [
-                    [0.005000012500022422, 7.227990390093038e-25],
-                    [-0.002500006250011211, -0.004330137844226556],
-                    [-0.002500006250011211, 0.004330137844226556],
-                    [-1.3476481215690672e-13, 2.891203383964549e-19],
-                ],
-                "currents2": [
-                    [-0.005000012500022422, -7.227990390093038e-25],
-                    [0.002500006250011211, 0.004330137844226556],
-                    [0.002500006250011211, -0.004330137844226556],
-                    [1.3476481215690672e-13, -2.891203383964549e-19],
-                ],
-            }
-        ],
-        "loads": [
-            {
-                "id": "load",
-                "phases": "abcn",
-                "currents": [
-                    [0.0050000125000625, 7.228026530113222e-25],
-                    [-0.002500006249963868, -0.004330137844254964],
-                    [-0.002500006249963868, 0.004330137844254964],
-                    [-1.3476372795473424e-13, 0.0],
-                ],
-            }
-        ],
-        "sources": [
-            {
-                "id": "vs",
-                "phases": "abcn",
-                "currents": [
-                    [-0.00500001250003125, -8.673617379884035e-19],
-                    [0.0025000062499482426, 0.004330137844227901],
-                    [0.0025000062499482426, -0.0043301378442279],
-                    [1.3476481215690672e-13, -2.891203383964549e-19],
-                ],
-            }
-        ],
-        "grounds": [
-            {"id": "ground", "potential": [0.0, 0.0]},
-        ],
-        "potential_refs": [
-            {"id": "pref", "current": [1.0842021724855044e-18, -2.891203383964549e-19]},
-        ],
-    }
+    # lp = LineParameters("test", z_line=10 * np.eye(2, dtype=complex))
+    # line = Line("line", bus0, bus1, phases=phases, parameters=lp, length=1.0, geometry=line_string)
+
+    # return ElectricalNetwork(
+    #     buses=[bus0, bus1],
+    #     branches=[line],
+    #     loads=[load],
+    #     sources=[vs],
+    #     grounds=[ground],
+    #     potential_refs=[pref],
+    # )
```

Comment:
You can delete the commented code in the fixture if you want

Reply:
Moved them to a script